lib/rgb_matrix/animation/config.ex

```diff
@@ -19,9 +19,9 @@ defmodule RGBMatrix.Animation.Config do
       RGBMatrix.Animation.HueWave.Config
       RGBMatrix.Animation.SolidReactive.Config
 
-  Configs should not be accessed or modified directly. Use the functions
-  `Xebow.get_animation_config/0` and `Xebow.update_animation_config/1` for
-  access and modification.
+  Configs should not be accessed or modified directly in an Animation module.
```

Comment:
Isn't this statement more accurate without `in an Animation module`? Shouldn't it always be accessed through the helper functions no matter which module that access happens in?

Reply:
Yeah...... I'm not sure what I was thinking at the time.